modules/assim.sequential/R/sda.enkf.R

```diff
@@ -83,19 +96,11 @@ sda.enkf <- function(settings, IC, prior, obs.mean, obs.cov, variables,
     print(paste("please make sure that the PEcAn interface is loaded for",model))
     stop()
   }
-  
-  ## split clim file
-  ## leaving in for sipnet
-  #   full.met <- settings$run$inputs$met$path
-  #   new.met  <- file.path(settings$rundir,basename(full.met))
-  #   file.copy(full.met,new.met)
-  #   met <- new.met#split.met.SIPNET(new.met)
-  
-  
+ 
   ###-------------------------------------------------------------------###
   ### perform initial set of runs                                       ###
   ###-------------------------------------------------------------------###  
-  X = IC
+  X = matrix(NA,as.numeric(settings$ensemble$size),length(settings$pft)) #why does X need to be here?
```

Comment:
I think that X is unnecessary because it gets created as a list in read.restart later. 

Right now, SIPNET is running off its default pools and the initial conditions are varied by sampling over all of the parameters during spin up. 

I'm not sure how to generalize this because linkages and other models don't use IC.